Documentation/codereview-policy.md

```diff
@@ -0,0 +1,45 @@
+### Code Review Policy
+
+#### How to review pull requests  
+
+This is a guide for core developers and contributors who want to help with code reviewing PRs on the fullnode.  
+We welcome anyone to help with code reviews, from typos to suggestions in design.   
+
+#### Why are code reviews important?  
+It's essential that PRs are properly code-reviewed.   
+Changes to the code, even small changes as incrementing an integer, can result in critical consensus bugs and loss of funds if not reviewed properly.  
+Some PRs are more severe than others in importance (for example changes to tests may not require the same level of checks as changes to the consensus code).  
+For that reason we have a PR review policy with 3 tags. A PR must be associated with a PR review policy.
+
+#### Requirements
+
+The requirement for each PR policy
+
+*pr-review-critical*
+- Code must be reviewed by at least two core devs
+- All tests must pass (Unit-test and Integration-tests) (run by author and the two reviewers)
+- All 4 chains (Bitcoin main/test, Stratis main/test) need to have been synced (to an agreed height) (run by author and 2 reviewers)
+
+*pr-review-high*
+- Code must be reviewed by at least two core devs
+- All tests must pass (Unit-test and Integration-tests) (run by author and the two reviewers)
+- All 4 chains (Bitcoin main/test, Stratis main/test) need to have been synced (to an agreed height)
+
+*pr-review-medium*
+- Code must be reviewed by at least two core dev (run by author and reviewr)
+- All tests must pass (Unit-test and Integration-tests) 
+
+*pr-review-low*
+- Code must be reviewed by at least one core dev
+- Unit-test must pass (automated CI tests)
+
```

Comment:
missing full stops on the above 2

Reply:
ooops - ignore that, I was comparing to the block at the bottom that all have full stops